File: src/test/java/no/nav/veilarboppgave/domene/PrioritetTest.java
package no.nav.veilarboppgave.domene;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class PrioritetTest {

    @Test
    void skal_utlede_korrekt_prioritetkode_basert_paa_tema_og_prioritet() {
        TemaDTO temaDTO = TemaDTO.OPPFOLGING;
        Prioritet pri = Prioritet.NORM;

        String kode = Prioritet.utledPrioritetKode(temaDTO, pri);
        assertEquals("NORM_OPP", kode);
    }
}

File: src/test/java/no/nav/veilarboppgave/domain/PrioritetTest.kt
package no.nav.veilarboppgave.domain

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class PrioritetTest {
    @Test
    fun skal_utlede_korrekt_prioritetkode_basert_paa_tema_og_prioritet() {
        val temaDTO = TemaDTO.OPPFOLGING
        val pri = Prioritet.NORM

        val kode = Prioritet.utledPrioritetKode(temaDTO, pri)
        Assertions.assertEquals("NORM_OPP", kode)
    }
}
